ansible/vars/production/production_public.yml

```diff
@@ -207,6 +207,7 @@ localsettings:
   GREENHOUSE_API_KEY: "{{ localsettings_private.GREENHOUSE_API_KEY }}"
   GMAPS_API_KEY: "{{ localsettings_private.GMAPS_API_KEY }}"
   GOOGLE_ANALYTICS_API_ID: "{{ localsettings_private.GOOGLE_ANALYTICS_API_ID }}"
+  GTM_ID: "{{ localsettings_private.GTM_ID }}"
```

Comment:
I"m guessing this isn't very private since it's in our hq git history

Reply:
Its also visible in the html source anyway :)